mmv1/api/resource.rb

```diff
@@ -147,6 +147,10 @@ module Properties
       # The Terraform resource id format used when calling #setId(...).
       # For instance, `{{name}}` means the id will be the resource name.
       attr_reader :id_format
+      # Sets the formats used to generate regex strings that match templated 
```

Comment:
`import_format` is an override, the id / self_link generally create decent ones. We should make it clear this is used to handle corner cases, not in most cases.

Reply:
Updated the phrasing, let me know if this sounds good

Reply:
SGTM!